diff -ur qbrew-0.4.1/src/beerxmlreader.cpp qbrew-0.4.1.new/src/beerxmlreader.cpp
--- qbrew-0.4.1/src/beerxmlreader.cpp	2008-05-25 21:21:04.000000000 -0500
+++ qbrew-0.4.1.new/src/beerxmlreader.cpp	2017-04-24 09:44:06.050285296 -0500
@@ -254,7 +254,6 @@
     }
     if (hasError()) {
         raiseError("Problem reading recipe");
-        return false;
     }
 
     // set grain use according to recipe type
