From 09082257b49fecf213522e84c42482115ea84878 Mon Sep 17 00:00:00 2001
From: Xiang, Haihao <haihao.xiang@intel.com>
Date: Tue, 24 Sep 2013 04:52:37 +0000
Subject: fix make dist

Signed-off-by: Xiang, Haihao <haihao.xiang@intel.com>
---
diff --git a/src/shaders/vme/Makefile.am b/src/shaders/vme/Makefile.am
index d3c2099..096cee4 100644
--- a/src/shaders/vme/Makefile.am
+++ b/src/shaders/vme/Makefile.am
@@ -64,11 +64,8 @@ EXTRA_DIST = \
 	$(INTEL_G75B)		\
 	$(INTEL_G7A)		\
 	$(INTEL_G7B)		\
-	$(INTEL_GEN6_ASM)	\
 	$(INTEL_GEN6_INC)	\
-	$(INTEL_GEN75_ASM)	\
 	$(INTEL_GEN75_INC)	\
-	$(INTEL_GEN7_ASM)	\
 	$(INTEL_GEN7_INC)	\
 	$(VME75_CORE)		\
 	$(VME_CORE)		\
--
cgit v0.9.0.2-2-gbebe
