diff -ur weather-2.0/weather.py weather-2.0.new/weather.py
--- weather-2.0/weather.py	2014-03-10 19:35:22.067287164 -0500
+++ weather-2.0.new/weather.py	2014-03-10 19:36:34.289787261 -0500
@@ -739,7 +739,7 @@
         datafile = datafiles[dataname][0]
         if datafile.endswith(".gz"):
             import gzip
-            stations.readfp( gzip.open(datafile) )
+            stations.readfp( gzip.open(datafile, "rt") )
         else:
             stations.read(datafile)
     else:
@@ -755,7 +755,7 @@
         datafile = datafiles[dataname][0]
         if datafile.endswith(".gz"):
             import gzip
-            zones.readfp( gzip.open(datafile) )
+            zones.readfp( gzip.open(datafile, "rt") )
         else:
             zones.read(datafile)
     else:
@@ -779,7 +779,7 @@
             datafile = datafiles[dataname][0]
             if datafile.endswith(".gz"):
                 import gzip
-                airports.readfp( gzip.open(datafile) )
+                airports.readfp( gzip.open(datafile, "rt") )
             else:
                 airports.read(datafile)
         else:
@@ -862,7 +862,7 @@
             datafile = datafiles[dataname][0]
             if datafile.endswith(".gz"):
                 import gzip
-                zctas.readfp( gzip.open(datafile) )
+                zctas.readfp( gzip.open(datafile, "rt") )
             else:
                 zctas.read(datafile)
         else:
@@ -917,7 +917,7 @@
             datafile = datafiles[dataname][0]
             if datafile.endswith(".gz"):
                 import gzip
-                places.readfp( gzip.open(datafile) )
+                places.readfp( gzip.open(datafile, "rt") )
             else:
                 places.read(datafile)
         else:
