diff -ur remmina-1.0.0_p20130723/remmina-plugins/rdp/rdp_plugin.c remmina-1.0.0_p20130723.new/remmina-plugins/rdp/rdp_plugin.c
--- remmina-1.0.0_p20130723/remmina-plugins/rdp/rdp_plugin.c	2013-07-22 22:11:14.000000000 -0500
+++ remmina-1.0.0_p20130723.new/remmina-plugins/rdp/rdp_plugin.c	2013-11-21 12:05:55.346924205 -0600
@@ -236,7 +236,7 @@
 		settings->LargePointerFlag = True;
 		settings->PerformanceFlags = PERF_FLAG_NONE;
 
-		rfi->rfx_context = rfx_context_new();
+		rfi->rfx_context = rfx_context_new(FALSE);
 	}
 
 	freerdp_channels_pre_connect(rfi->channels, instance);
