diff -ur Remmina-9999/remmina/CMakeLists.txt Remmina-9999.new/remmina/CMakeLists.txt
--- Remmina-9999/remmina/CMakeLists.txt	2015-04-21 11:49:20.140974910 -0500
+++ Remmina-9999.new/remmina/CMakeLists.txt	2015-04-21 11:49:41.348796522 -0500
@@ -176,7 +176,6 @@
 
 add_subdirectory(icons)
 add_subdirectory(desktop)
-add_subdirectory(external_tools)
 add_subdirectory(ui)
 
 install(TARGETS remmina DESTINATION ${CMAKE_INSTALL_BINDIR})
