diff -ur cinnamon-screensaver-1.8.0/configure.ac cinnamon-screensaver-1.8.0.new/configure.ac
--- cinnamon-screensaver-1.8.0/configure.ac	2013-06-30 12:39:38.860970507 -0500
+++ cinnamon-screensaver-1.8.0.new/configure.ac	2013-06-30 12:40:26.543664509 -0500
@@ -768,7 +768,6 @@
 src/cinnamon-screensaver.desktop.in
 data/Makefile
 data/org.cinnamon.ScreenSaver.service
-files/Makefile
 ])
 
 echo "
Only in cinnamon-screensaver-1.8.0: files
diff -ur cinnamon-screensaver-1.8.0/Makefile.am cinnamon-screensaver-1.8.0.new/Makefile.am
--- cinnamon-screensaver-1.8.0/Makefile.am	2013-06-30 12:39:38.867970469 -0500
+++ cinnamon-screensaver-1.8.0.new/Makefile.am	2013-06-30 12:40:11.285762472 -0500
@@ -8,7 +8,6 @@
 	src			\
 	po 			\
 	data			\
-    files       \
 	$(NULL)
 
 EXTRA_DIST =			\
