diff -ur mce-test-20120424.old/cases/function/hwpoison/Makefile mce-test-20120424/cases/function/hwpoison/Makefile
--- mce-test-20120424.old/cases/function/hwpoison/Makefile	2013-09-04 14:34:12.314127522 -0500
+++ mce-test-20120424/cases/function/hwpoison/Makefile	2013-09-04 14:34:31.393952354 -0500
@@ -10,7 +10,7 @@
 install: all
 	cp ttranshuge ../../../bin/
 
-tinjpage: LDFLAGS += -lpthread
+tinjpage: LDFLAGS += -pthread
 
 clean:
 	rm -f ${EXE} ${OBJ}
