diff -ur iftop-1.0pre3/configure.in iftop-1.0pre3.new/configure.in
--- iftop-1.0pre3/configure.in	2014-01-07 10:09:38.459108009 -0600
+++ iftop-1.0pre3.new/configure.in	2014-01-07 10:12:35.709503563 -0600
@@ -350,19 +350,23 @@
 dnl AC_SEARCH_LIBS....
 dnl
 
-AC_MSG_CHECKING([for a curses library containing mvchgat])
-oldLIBS=$LIBS
-for curseslib in ncursesw curses ncurses ; do
-    LIBS="$oldLIBS -l$curseslib"
-    AC_TRY_LINK([
+PKG_CHECK_MODULES([NCURSES], [ncursesw], [foundcurseslib="$NCURSES_LIBS"], [
+    PKG_CHECK_MODULES([NCURSES], [ncurses], [foundcurseslib="$NCURSES_LIBS"], [
+        AC_MSG_CHECKING([for a curses library containing mvchgat])
+        oldLIBS=$LIBS
+        for curseslib in ncursesw curses ncurses ; do
+            LIBS="$oldLIBS -l$curseslib"
+            AC_TRY_LINK([
 #include <$curseslib.h>
-        ], [
-        mvchgat(0, 0, 1, A_REVERSE, 0, NULL)
-        ], [
-        foundcurseslib=$curseslib
-        break
-        ])
-done
+                 ], [
+                 mvchgat(0, 0, 1, A_REVERSE, 0, NULL)
+                 ], [
+                 foundcurseslib=-l$curseslib
+                 break
+                 ])
+        done
+    ])
+])
 
 if test x$foundcurseslib = x ; then
     AC_MSG_RESULT([none found])
diff -ur iftop-1.0pre3/Makefile.am iftop-1.0pre3.new/Makefile.am
--- iftop-1.0pre3/Makefile.am	2014-01-07 10:09:38.460107976 -0600
+++ iftop-1.0pre3.new/Makefile.am	2014-01-07 10:10:36.155283596 -0600
@@ -19,6 +19,8 @@
 		 addrs_ioctl.c addrs_dlpi.c dlcommon.c \
 		 stringmap.c cfgfile.c vector.c
 
+iftop_LDADD = $(NCURSES_LIBS)
+
 #iftop_dump_SOURCES = counter_hash.c hash.c iftop-dump.c  \
 #                 options.c \
 #                 util.c \
